pkg/trait/route.go

```diff
@@ -29,10 +29,16 @@ import (
 )
 
 type routeTrait struct {
-	BaseTrait `property:",squash"`
-	Auto      *bool  `property:"auto"`
-	Host      string `property:"host"`
-	service   *corev1.Service
+	BaseTrait                        `property:",squash"`
+	Auto                             *bool  `property:"auto"`
+	Host                             string `property:"host"`
+	TLSTermination                   string `property:"tls-termination"`
+	TLSCertificate                   string `property:"tls-certificate"`
+	TLSKey                           string `property:"tls-key"`
+	TLSCACertificate                 string `property:"tls-cacertificate"`
```

Comment:
`property:"tls-ca-certificate"` may be more legible.

Reply:
oh yes, will update

Reply:
@astefanutti should be better now